### PR TITLE
[DO NOT MERGE] Changelog-0.16.0

### PR DESCRIPTION
**Pull request by Manuel-K**
Initial Version

### CODE REVIEW

**Comment by lynxlynxlynx** [comment]
the pages are from the milestone list?

**Comment by Manuel-K** [comment]
Yes, to make it easier to find the issue during editing.

**Comment by Manuel-K** [comment]
We have to do something about the “It is easier to find, too.”.

File: Changelog-0.16.0.md [hunk 1]
```diff
@@ -128,6 +128,7 @@ __Full Changelog__:
  - Enhancement: Add “IMP Quiz Honest Answers” mini mod (#303)
  - Enhancement: Simplify moving to the next sector in tactical view (windowed mode) (#314)
  - Enhancement: Add “Generous Rebels” mini mod (#305)
+ - Maintenance: Explicitly initialize MovementMode (#635)
```
Comment (lynxlynxlynx) [inline Changelog-0.16.0.md]: This is a vanilla fix and change.

Comment (Manuel-K) [reply Changelog-0.16.0.md]: Are you sure? I would classify it as: 1/3 vanilla code quality improvements, 2/3 “follow up of bug fix” code quality improvements.

**Pull request merged**